res/controllers/Denon-MC7000-scripts.js

```diff
@@ -668,13 +680,11 @@ MC7000.wheelTurn = function(channel, control, value, status, group) {
     } else {
         if (MC7000.shift[deckOffset]) {
             // While Shift Button pressed -> Search through track
-            var jogSearch = 300 * adjustedSpeed / MC7000.jogWheelTicksPerRevolution;
+            var jogSearch = 100 * adjustedSpeed;
```

Comment:
What is the meaning of `100`? Is this a global scale factor? Please add a comment or make it a global constant. Is `MC7000.jogWheelTicksPerRevolution` unused now?

Reply:
There is no real effect using the `MC7000.jogWheelTicksPerRevolution` for search through track. It now only considers the Midi values and uses the multiplier 100 to quickly navigate through the track. I gonna add a note behind the line ...
The variable `MC7000.jogWheelTicksPerRevolution` is still used inside `engine.scratchEnable`